ntbs-service/wwwroot/css/header.scss

```diff
@@ -130,12 +130,10 @@ body:not(.js-enabled) {
 .header-logo {
     padding: 0 40px 5px 0;
     margin: 0;
-    width: 200px;

```

Comment:
This logo is much longer than the old one - the aspect ratio is different making it appear squished vertically, so I have removed the predefined width so while longer this logo is the same height as the old one was. This doesn't affect accessibility for smaller computer window sizes as the NTBS text drops to a new row on resize. However, for mobile devices (is this even a consideration we need to make?) the logo is long enough to overhang the menu button. Perhaps a media breakpoint to make the logo smaller on very small screen widths?

Reply:
Comparing the logos I'm surprised that this would make much of a difference (on my mobile there's quite a lot of space between the logo and menu button) so may only affect some mobile devices. A media query definitely seems reasonable.

That said, I'm a bit confused about the Menu button (I'd never seen it before as it only appears on mobile displays) - it looks like it's meant to be giving you the option to show/hide the navigation menu, but in practice it doesn't seem to do anything. @jordan-carlton Was it functional for you at all? @aptemus Do you know anything about this?

Reply:
yes as far as I know the menu button should collapse/expand the navigation menu. It isn't working for me on Edge or Chome actually on my mobile device, or via developer tools.

Reply:
I've done some digging and this was actually broken all the way back on NTBS-359! I've got an idea of what went wrong and I think we should try to fix it - @jordan-carlton give me a call when you're available and we can discuss

Reply:
I've added a css class to the mobile menu which was previously removed which should hopefully the issue with the mobile menu button not working